brianlong/www.validators.app
# 183730528 - add pythnet to services

#### What's this PR do?
PR updates services with pythnet.

#### How should this be manually tested?
Changes have impact on the following services:
SolanaRpcClient
ValidatorCheckActiveService
ValidatorSoftwareVersion

You can run them using the console in order to check if any fails. The database can be required to be populated with pythnet data.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183730528)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
